deps/v8/gypfiles/v8_external_snapshot.gypi

```diff
@@ -61,24 +62,31 @@
               ['v8_os_page_size!=0', {
                 'mksnapshot_flags': ['--v8_os_page_size', '<(v8_os_page_size)'],
               }],
+              ['v8_enable_embedded_builtins="true"', {
+                # 'embedded_builtins_snapshot_src': [ "$target_gen_dir/embedded${suffix}.cc" ],
```

Comment:
is this supposed to be commented?

Reply:
We don't build or test with `external_snapshot` so I was lazy.
https://github.com/nodejs/node/blob/ae461f52330710858f1282bfacdd2da6e0145dd9/common.gypi#L39-L40
(no mention in `configure.py` either)